steward/tests/acceptance.rs

```diff
@@ -30,65 +35,436 @@ use steward::config::StewardConfig;
 /// invocations as `cargo test` executes tests in parallel by default.
 pub static RUNNER: Lazy<CmdRunner> = Lazy::new(|| CmdRunner::default());
 
-/// Use `StewardConfig::default()` value if no config or args
+/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_no_args() {
+fn eth_keys_add() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
+    let config_dir = TempDir::new("test_key")?;
+    let config_file_path = config_dir.path().join("config.toml");
+    let mut f = File::create(config_file_path.clone())?;
+    let configu = StewardConfig {
+        cellars: StewardConfig::default().cellars,
+        ethereum: StewardConfig::default().ethereum,
+        keys: KeysConfig {
+            keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+        server: StewardConfig::default().server,
+        gravity: StewardConfig::default().gravity,
+        cosmos: StewardConfig::default().cosmos,
+        metrics: StewardConfig::default().metrics,
+    };
+    f.write_all(toml::to_string(&configu).unwrap().as_bytes())?;
     let mut runner = RUNNER.clone();
-    let mut cmd = runner.arg("start").capture_stdout().run();
-    cmd.stdout().expect_line("Hello, world!");
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "eth",
+            "add",
+            "sha",
+        ])
+        .capture_stdout()
+        .run();
+    // Check that command executes without error.
     cmd.wait().unwrap().expect_success();
+
+    f.sync_all()?;
+
+    Ok(())
 }
 
-/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_with_args() {
+fn eth_keys_delete() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
+    let key_file_path = key_dir.path().join("ethkey.pem");
+    let mut f = File::create(key_file_path.clone())?;
+    f.write_all(
+        b"-----BEGIN PRIVATE KEY-----
```

Comment:
Even though it doesn't appear to be read, we're still writing this file to disk incorrectly if we aren't including `\n\` at the end. We're also repeating this string in a lot of places, it would be good to define it in one variable or create a function to write this file to a specified path.

Reply:
This should resolve it 935d9bfdb055e6c8860324b598eb0fd4e53330a2